src/api/productReviewAPI.ts

```diff
@@ -0,0 +1,17 @@
+import { ProductReviewType } from '@/types/ProductReviewTypes';
+
+export const getProductReviews = async (productId: string): Promise<ProductReviewType> => {
+  try {
+    const res = await fetch(
+      `${process.env.NEXT_PUBLIC_KEYDEUK_API_BASE_URL}/api/v1/reviews?productId=${productId}&sort=createdAt&page=0&size=10`,
+      {
+        cache: 'no-store',
+      },
+    );
+    const result = await res.json();
+
+    return result.data;
```

Comment:
이런 식으로 하는 게 더 간결할 것 같아요~
```suggestion
    const { data }= await res.json();

    return data;
```